### ISSUE TITLE
Keyboard shortcuts with arrows do not work

### ISSUE THREAD

**Issue by rrgeorge** [blinksh/blink#1281]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Describe the bug
Shortcuts (Cmd+Shift+⬅️  and Cmd+Shif+➡️) to change tabs dont work with latest TestFlight build

**Comment by carloscabanero** [comment]
Fixed as well ;)

**Comment by yury** [comment]
Not working for me :(

**Comment by carloscabanero** [comment]
Make sure there is just a single shortcut for that action. We do not allow multiple anymore.

**Comment by olober** [comment]
I reseted all shortcuts. I try to setup next/prev tab:
1st attempt:
1) next/prev tab: OPT-CMD-[ or ] = works
2) next/prev tab (cycling): OPT-CMD-arrows = does not work

2nd try: removed OPT-CMD-[ or ], next/prev tab cycling with OPT-CMD-arrows does not work.

3rd try: change of OPT-CMD-arrow to next/prev tab (without cycling) = it works.

I tried a few other combinations. It looks like the "next/prev tab cycling" commands DOES NOT work no matter what keyboard combo I set.

iOS 15.0.2, iPad Pro with Magic Keyboard
